app/experimenter/visualization/tests/api/test_views.py

```diff
@@ -36,6 +36,16 @@ def test_analysis_results_view_200(self, lifecycle, mock_exists):
         )
         self.assertEqual(response.status_code, 200)
 
+        # test with results_data object
+        experiment.results_data = {"v1": {}}
+        experiment.save()
+
+        response = self.client.get(
+            reverse("visualization-analysis-data", kwargs={"slug": experiment.slug}),
+            **{settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
```

Comment:
This doesn't need to be a separate step, just set the results data on the create and assert the contents after the response code?

Reply:
I left it in to test the `None` case, but updated now to check the contents for both.

Reply:
I still would've made those two separate tests, but not a big deal 👍